src/main/java/Duke.java

```diff
@@ -1,10 +1,95 @@
+                int index = Integer.parseInt(instrSplit[1]);
+                task = taskList.get(index - 1);
+                task.setDone();
+                System.out.println(line + indent + "Nice! I've marked this task as done:");
+                System.out.println(indent + "  " + task + "\n" + line);
+            } else if (command.compareTo("unmark") == 0) {
+                int index = Integer.parseInt(instrSplit[1]);
+                task = taskList.get(index - 1);
+                task.setUndone();
+                System.out.println(line + indent + "OK, I've marked this task as not done yet:");
+                System.out.println(indent + "  " + task + "\n" + line);
+            } else {
+                System.out.print(line);
+                if (command.compareTo("delete") == 0) {
+                    int index = Integer.parseInt(instrSplit[1]);
+                    task = taskList.get(index - 1);
+                    taskList.remove(index - 1);
+                    System.out.println(indent + "Noted. I've removed this task:");
+                    System.out.println(indent + "  " + task);
+                } else if (command.compareTo("todo") == 0) {
+                    if (instrSplit.length == 1) {
+                        System.out.println(indent + "☹ OOPS!!! The description of a todo cannot be empty.\n" + line);
+                        continue;
+                    }
+                    System.out.println(indent + "Got it. I've added this task:");
+                    Todo todo = new Todo(instruction.substring(5));
+                    taskList.add(todo);
+                    System.out.println(indent + "  " + todo);
+                } else if (command.compareTo("deadline") == 0) {
+                    if (instrSplit.length == 1) {
+                        System.out.println(indent + "☹ OOPS!!! The description of a deadline cannot be empty.\n" + line);
+                        continue;
+                    }
+                    System.out.println(indent + "Got it. I've added this task:");
+                    Deadline deadline = new Deadline(instruction.substring(9));
+                    taskList.add(deadline);
+                    System.out.println(indent + "  " + deadline);
+                } else if (command.compareTo("event") == 0) {
+                    if (instrSplit.length == 1) {
+                        System.out.println(indent + "☹ OOPS!!! The description of an event cannot be empty.\n" + line);
+                        continue;
+                    }
+                    System.out.println(indent + "Got it. I've added this task:");
+                    Event event = new Event(instruction.substring(6));
+                    taskList.add(event);
+                    System.out.println(indent + "  " + event);
+                } else {
+                    System.out.println(indent + "☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n" + line);
+                    continue;
+                }
```

Comment:
You may consider adding some exception handling in case the user puts in an invalid input.